misc/python/materialize/mzcompose/composition.py

```diff
@@ -490,6 +490,10 @@ def workflow_default(c: Composition):
             ]
 
             if isinstance(e, CommandFailureCausedUIError):
+                if e.stderr is not None:
+                    # This is to avoid that captured stderr is missing in the logs.
+                    print(e.stderr)
```

Comment:
I think for consistency with other tools we might want to print this out to stderr again.